src/security/forward.h

```diff
@@ -77,9 +81,19 @@ typedef struct gnutls_x509_crt_int Certificate;
 typedef struct notls_x509 Certificate;
 #endif
 
+#if USE_OPENSSL || USE_GNUTLS || IN_TESTHEADERS
```

Comment:
Why do we need to make CtoCpp1() definition conditional in this PR? CtoCpp1() does not seem to depend on `USE_OPENSSL || USE_GNUTLS || IN_TESTHEADERS`.

Reply:
Right. Making this unconditional

Reply:
done in 289fc1e0e5